Marlin/SanityCheck.h

```diff
@@ -361,8 +367,14 @@
  */
 #ifdef SNMM
   #error "SNMM is now MK2_MULTIPLEXER. Please update your configuration."
-#elif ENABLED(MK2_MULTIPLEXER) && DISABLED(ADVANCED_PAUSE_FEATURE)
-  #error "ADVANCED_PAUSE_FEATURE is required with MK2_MULTIPLEXER."
+#elif ENABLED(MK2_MULTIPLEXER)
+  #if ENABLED(MIXING_EXTRUDER)
+    #error "MK2_MULTIPLEXER is incompatible with MIXING_EXTRUDER."
+  #elif !defined(E_MUX0_PIN)
```

Comment:
Pins are allowed to be defined as `-1` to indicate no pin, so be sure to use `PIN_EXISTS(E_MUX0)` for this and other pin tests.